libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxService.java

```diff
@@ -0,0 +1,227 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import com.google.gson.GsonBuilder;
+
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Mapbox specific services used internally within the SDK. Subclasses must implement baseUrl and
+ * initializeCall.
+ *
+ * @param <T> Type parameter for response.
+ * @param <S> Type parameter for service interface.
+ * @since 1.0.0
+ */
+public abstract class MapboxService<T, S> {
```

Comment:
> Class 'MapboxService' is never used

Is this something that we're taking from MAS and therefore should be removed from here (Nav side)? Am I missing something?

Reply:
oops, good catch. Meant to remove that